arch/riscv/configs/defconfig

```diff
@@ -240,3 +240,5 @@ CONFIG_RCU_EQS_DEBUG=y
 # CONFIG_FTRACE is not set
 # CONFIG_RUNTIME_TESTING_MENU is not set
 CONFIG_MEMTEST=y
+CONFIG_EFI=y
```

Comment:
I know this was done in an old commit.
But we should revert this change and have a separate `cartesi_defconfig` instead.

I'll add an issue for this.

Reply:
The PR is good as is.
I think we can do this change at a later moment.